ziti/cmd/ascode/exporter/exporter.go

```diff
@@ -119,7 +119,7 @@ func (exporter *Exporter) Init(out io.Writer) error {
 	var err error
 	exporter.client, err = exporter.loginOpts.NewMgmtClient()
 	if err != nil {
-		log.Fatal(err)
+		panic(err)
```

Comment:
the functions already indicate they return errors, probably even better to just return instead?